app/services/conferences/creation_service.rb

```diff
@@ -34,11 +34,6 @@ def sanatize_params(params)
       params[:online] = true
     end
 
-    if params[:online] == true
```

Comment:
Maybe you should do this instead then:
```rb
if params[:online] == true && params[:city].blank?
```

Otherwise you might have the 2 keys "country" / "city" as empty string in the JSON.